system_tests/fees_test.go

```diff
@@ -2,8 +2,10 @@
 // For license information, see https://github.com/nitro/blob/master/LICENSE
 
 // these tests seems to consume too much memory with race detection
-//go:build !race
-// +build !race
+// Test randomly fails with L1 gas price estimate should tend toward the basefee
+// so skipping locally, but running on CI
+//go:build !race && feestest
+// +build !race,feestest
```

Comment:
let's have a more generic tag that we can use for other things.. like nolocal / cionly / etc..

Reply:
Done.